screens/product-listing-page/source/style.css

```diff
@@ -0,0 +1,158 @@
+
+
+/* Mobile Devices */
+    /* Utils */
+.pos-fix-left{
+    position:fixed;
+    top:0;
+    bottom: 0;
+    width: 60%;
+    background-color: var(--background-theme);
+    z-index: 9;
+    
+    
+    word-break: break-all;
+
+}
+
+
+
+
+
+
+.heart-icon{
```

Comment:
You should consider using a code formatter like Prettier for consistent spacing and clean code.

Reply:
Sure, thanks for the suggestion